Data/ByteString/Builder/ASCII.hs

```diff
@@ -105,7 +105,7 @@ import           GHC.Types   (Int(..))
 # if __GLASGOW_HASKELL__ < 611
 import GHC.Integer.Internals
 # else
-import GHC.Integer.GMP.Internals
+import GHC.Integer.GMP.Internals -- for S# constructor
```

Comment:
why don't you express the comment as an explicit import list instead?

Reply:
done